cmd/internal/cli/push.go

```diff
@@ -93,6 +93,7 @@ var PushCmd = &cobra.Command{
 			if err := oras.UploadImage(file, ref, ociAuth); err != nil {
 				sylog.Fatalf("Unable to push image to oci registry: %v", err)
 			}
+			sylog.Infof("Upload complete.")
```

Comment:
Whats the `.` for? I dont think we use any _dots_ in our other Info messages...

Reply:
Not necessary, ill change that